content/enterprise_influxdb/v1.9/about-the-project/third-party.md

```diff
@@ -0,0 +1,49 @@
+| [goconvey](https://github.com/glycerine/goconvey) | [MIT](https://opensource.org/licenses/MIT)  | Statically linked |
+| [go-immutable-radix](https://github.com/hashicorp/go-immutable-radixhttps://github.com/hashicorp/go-immutable-radix) |  [Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0) | Statically linked |
+| [going](https://github.com/markbates/going) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |Statically linked|
+| [golang-lru](https://github.com/hashicorp/golang-lru) |[Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0)  | Statically linked |
+| [go-msgpack](https://github.com/hashicorp/go-msgpack) | [BSD 3-Clause \*](https://opensource.org/licenses/BSD-3-Clause) | Statically linked |
+| [go-metrics](https://github.com/armon/go-metrics) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |
+| [go-uuid](https://github.com/hashicorp/go-uuid) | [Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0) | Statically linked |
+| [handlers](https://github.com/gorilla/handlershttps://github.com/gorilla/handlers) | [BSD 2-Clause \*](https://opensource.org/licenses/BSD-2-Clause) | Statically linked |
+| [jose2go](https://github.com/dvsekhvalnov/jose2go) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |
+| [ldap](https://github.com/go-ldap/ldap) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |
+| [ldapserver](https://github.com/mark-rushakoff/ldapserverhttps://github.com/mark-rushakoff/ldapserver) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |Statically linked|
+| [mux](https://github.com/gorilla/mux) | [BSD 2-Clause \*](https://opensource.org/licenses/BSD-2-Clause) | Statically linked |
+| [pkcs7](https://github.com/fullsailor/pkcs7) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |
+| [pretty](https://github.com/kr/pretty) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |Statically linked|
+| [raft](https://github.com/hashicorp/raft) | [Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0)  | Statically linked |Statically linked|
+| [raft-boltdb](https://github.com/hashicorp/raft-boltdb) | [Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0)  | Statically linked |Statically linked|
+| [sqlx](https://github.com/jmoiron/sqlxhttps://github.com/jmoiron/sqlx) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |Statically linked|
+| [text](https://github.com/kr/text) | [MIT](https://opensource.org/licenses/MIT) | Statically linked |Statically linked|
+| [yamux](https://github.com/hashicorp/yamux/)| [Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0) | Statically linked |Statically linked|
+
+\* BSD 3-Clause is also known as the "New" or "Modified" BSD License. BSD 2-Clause is also known as the "Simplified" or "Free" BSD License.
```

Comment:
line 49 looks like a stray?

Reply:
Remove note about BSD licenses, now we're linked to the license.